src/porepy/numerics/contact_mechanics/contact_conditions.py

```diff
@@ -116,21 +120,39 @@ def discretize(self, g_h, g_l, data_h, data_l, data_edge):
         # sign of the second mortar grid, and then sum the displacements on the
         # two sides (which is really a difference since one of the sides have
         # its sign switched).
-        displacement_jump_global_coord = (
+        # The tangential displacements are relative to the initial state, which in the
+        # transient case equals the previous time step.
+        previous_displacement_iterate = data_edge[pp.STATE]["previous_iterate"][
+            self.mortar_displacement_variable
+        ]
+        previous_displacement_time = data_edge[pp.STATE][
+            self.mortar_displacement_variable
```

Comment:
is self.mortar_displacement_keyword a better name?

Reply:
I don't think so, seeing as this is the stored variable.